language/stdlib/transaction_scripts/register_validator.move

```diff
@@ -1,5 +1,4 @@
 script {
-    use 0x1::LibraSystem;
```

Comment:
I think this is now identical to `rotate_validator_config`--we can probably kill it?